src/app/components/sunburst/sunburst.tsx

```diff
@@ -15,18 +30,33 @@ export function Sunburst(props: {
       if (props.data) {
         if (props.sortByVersion) {
           chart.appendChild(
-            sunburst(props.data, (a, b) => {
-              const vA = coerce(a.data.name);
-              const vB = coerce(b.data.name);
-              if (!vA || !vB) {
-                return a.value! - b.value!;
-              } else {
-                return gt(vA, vB) ? -1 : 1;
-              }
+            sunburst({
+              data: props.data,
+              sortComparator: (a, b) => {
+                const vA = coerce(a.data.name);
+                const vB = coerce(b.data.name);
+                if (!vA || !vB) {
+                  return a.value! - b.value!;
+                } else {
+                  return gt(vA, vB) ? -1 : 1;
+                }
+              },
+              selectionUpdated: (selection) => {
+                setSelectedVersion(selection);
+                props.onVersionChange(selection);
+              },
```

Comment:
Lets pull `data` and `selectionUpdated` out since they are the same between the 2 appendChild calls. Something like:

```typescript
const sunburstOptions = {
  data: props.data,
  selectionUpdated: (selection) => {
    setSelectedVersion(selection);
    props.onVersionChange(selection);
  }
}

if (props.sortByVersion) {
  sunburstOptions.sortComparator = sortByVersion
}
chart.appendChild(sunburst(sunburstOptions))
```